src/style.css

```diff
@@ -0,0 +1,53 @@
+* {
+  margin: 0;
+}
+
+.star {
+  width: 100px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  margin-right: 4px;
+  justify-content: space-between;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("images/star.svg");
+}
+
+.stars--1 > :first-child {
+  display: flex;
+  background-repeat: no-repeat;
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 > :nth-child(n+0):nth-last-child(n+4) {
+  display: flex;
+  background-repeat: no-repeat;
+  background-image: url("images/star-active.svg");
+}
```

Comment:
1. Don't repeat yourself, combine selectors for same rules.
2. Are you sure you need display flex here?
3. Background-repeat rule is the same in stars__star. You don't need to overwrite it.